test/TestConfig/settings.mk

```diff
@@ -58,6 +58,28 @@ else
 export JAVA_VERSION:=$(JAVA_VERSION)
 endif
 
+ifndef JAVA_IMPL
```

Comment:
I would prefer there is `ifndef JVM_VERSION` block. That is saying user can also define his own JVM_VERSION.

Reply:
Done!